docs/maintainers/good_first_issues_guidelines.md

```
@@ -5,223 +5,146 @@ This document defines what we **do** and **do not** consider a *Good First Issue
-  - [Small, Focused Source Changes](#-small-focused-source-changes)
-  - [Typing Improvements](#-typing-improvements)
-  - [Refactors of Existing Examples](#-refactors-of-existing-examples)
-  - [Documentation Improvements](#-documentation-improvements)
-  - [Print and Output Clarity](#️-print-and-output-clarity-examples-only)
-  - [Functional Improvements to Examples](#️-functional-improvements-to-examples)
-  - [Test Improvements](#-test-improvements-additive-only)
-- [What We Do NOT Consider Good First Issues](#-what-we-do-not-consider-good-first-issues)
+- [What We Consider Good First Issues](#what-we-consider-good-first-issues)
+  - [Small, Focused Source Changes](#small-focused-source-changes)
+  - [Typing Improvements](#typing-improvements)
+  - [Refactors of Existing Examples](#refactors-of-existing-examples)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)